tests/src/rules/no-unlocalized-strings.test.ts

```diff
@@ -16,6 +16,10 @@ const ruleTester = new RuleTester({
   },
 })
 
+const upperCaseRegex = '^[A-Z_-]+$'
```

Comment:
Could you extend the regex to allow numbers?

I created a pull request here https://github.com/lingui/eslint-plugin/pull/79

Reply:
Good, thank you